lib/is-run-out.js

```diff
@@ -1,11 +1,12 @@
 const Opcode = require('bsv/lib/opcode');
 
 const isRunOut = (script) => {
-	return !!(
-		script.chunks[0].opcodenum === Opcode.OP_0 &&
+	if (script.chunks[0].opcodenum === Opcode.OP_0 &&
 		script.chunks[1].opcodenum === Opcode.OP_RETURN &&
-		script.chunks[2].buf.toString('utf8') === 'run'
-	);
+		script.chunks[2].buf.toString('utf8') === 'run') {
+		return script.chunks[4].buf ? script.chunks[4].buf.toString('utf8') : 'run';
+	}
+	else { return '' }
```

Comment:
`else` here is unnecessary since we're returning inside the if statement. Instead we can just `return false` after the if statement